backend/worker/requirements.txt

```diff
@@ -5,3 +5,5 @@ cryptography==2.9
 pytest==6.0.1
 scrapy==2.3.0
 dnstwist==20201228
+whois==0.9.7
+dnspython==2.1.0
```

Comment:
Do we need this line? I think we can remove it because it may be dependency of dnstwist already

Reply:
I noticed more domains 

> Looks good! I'm a bit confused as to the plan, though. It seems like the python whois module is working -- why do we need to use the npm whois now?

We can use the python module. I was just thinking it would be better to use a node library instead of spawning python, but I guess it doesn't matter?

Reply:
And for the requirements.txt:
<img width="878" alt="Screen Shot 2021-05-27 at 11 47 06 AM" src="https://user-images.githubusercontent.com/79927030/119857291-6ec11a00-bee1-11eb-8947-549fdc401f79.png">

I noticed adding dnspython returned more domains and whois won't run without it.